2D3D_UnityProject/Assets/Scripts/Utility/UI/DialogueLine.cs

```diff
@@ -46,12 +58,37 @@ public class DialogueLine : MonoBehaviour
     [Range(0, 1)]
     private float soundBlipSpeed = .5f;
 
+    /// <summary>
+    /// reference to parent canvas object
+    /// </summary>
+    public GameObject canvasObj;
+
     // Update is called once per frame
     void Update()
     {
-        if(printDialogue == null) {
+        if (printDialogue == null)
+        {
             printDialogue = StartCoroutine(PrintDialogueCoroutine());
         }
+
+        // if the player presses space while talking
+        if (Input.GetKeyDown(KeyCode.Space))
+        {
+            //if the dialogue has finished printing, exit the conversation
+            if (FinishedPrinting())
+            {
+                printDialogue = null;
+                dialogueText.text = "";
+                canvasObj.SetActive(false);
+                LiarGameManager.Instance().player.GetComponent<PlayerController>().enabled = true;

```

Comment:
This is why we should have a more generic DialogueManager class - our Dialogue code now directly references the Liars puzzle. 

(not to cast shade, i know we're in the prototype phase still, just wanna catch this stuff as early as possible)
we should rework the dialogue code before we need to implement more dialogue in the game